### DISCUSSION TITLE
Unable to create subscription for FBA_INVENTORY_AVAILABILITY_CHANGES for UK Marketplace

### DISCUSSION THREAD

**Discussion by aman863** [amzn/selling-partner-api-models#2826] [General]
Hi,

I'm able to create subscriptions for ANY_OFFER_CHANGE, PRICING_HEALTH and am receiving a subscription ID in the response. But I'm unable to create a subscription for FBA_INVENTORY_AVAILABILITY_CHANGES. I've authorization from Seller central UK and the region is eu-west-1. It returns with error 403 Forbidden and the error message "Access to requested resource is denied." with no details provided. 

Please help me out here

**Comment by stefnats** [comment]
Are you sure you have all the necessary roles, i.e. Amazon Fulfillment?

Refer to this page: https://developer-docs.amazon.com/sp-api/docs/roles-in-the-selling-partner-api

My bet, without knowing anything specific, is that you're missing that role. After you change roles, you need to redo the authorization before you can try again.

Hope this helps, please keep me posted.

**Comment by aman863** [comment]
Hey @stefnats , Yes I was missing the role only. Thanks a lot for the help.
